tests/common

```diff
@@ -0,0 +1,46 @@
+#!/bin/ksh
+set -e 
+failed=0
+
+workdir="$PWD"
+tmpdir=$(mktemp -d -p $workdir)
+
+port=13245
+www=$(mktemp -d -p $tmpdir)
+shellweb_cmd="${workdir}/shellweb -d $www -p $port"
+
+test_name=${0##*/test-}
+
+run_shellweb(){
+	$shellweb_cmd &
+	sleep 0.1
```

Comment:
`0.1` would not be enough on slow/busy machines... Please increase to `0.5` or ever `1`.

Better solution, of course, would be to wait for connection to be opened with netstat, but the current solution is Good Enough(TM).